src/TimerDialog.cpp

```diff
@@ -0,0 +1,250 @@
+#include "TimerDialog.h"
+#include <QChar>
+#include <QFileDialog>
+#include <QDir>
+#include <QUrl>
+#include <QSpacerItem>
+#include <QHBoxLayout>
+#include <QVBoxLayout>
+#include "brewtarget.h"
+#include <QMessageBox>
+
+TimerDialog::TimerDialog(TimerListDialog *parent, BoilTime* bt) :
+    QDialog(parent),
+    mainTimer(parent),
+    paletteOld(),
+    paletteNew(),
+    oldColors(true),
+    ui(new Ui::TimerDialog),
+    boilTime(bt),
```

Comment:
This causes a compiler error when `NO_QTMULTIMEDIA` is defined due to the trailing comma. Move comma to the beginning of the line, e.g.

    , mediaPlayer(...)
    , playlist(...)